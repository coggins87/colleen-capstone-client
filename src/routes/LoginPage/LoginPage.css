.LoginPage {
  display: flex;
  flex-direction: column;
  margin: auto;
  padding-top: 5%;
}

.page_header {
  font-family: "Nanum Myeongjo", serif;

}