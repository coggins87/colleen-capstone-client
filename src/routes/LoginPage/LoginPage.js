import React from 'react';
import LoginForm from '../../components/LoginForm/LoginForm'
import ApiContext from '../../context/ApiContext';

class LoginPage extends React.Component{
  static defaultProps = {
    location: {},
    history: {
      push: ()=>{}
    },
    
  }
 static contextType = ApiContext

  render(){
    return(
      <section className="LoginPage">
        <h2>Login</h2>
      <LoginForm onLoginSuccess={this.handleLoginSuccess} />
      </section>
    )
  }
}

export default LoginPage;