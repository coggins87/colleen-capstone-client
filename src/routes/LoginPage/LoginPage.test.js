import React from 'react';
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import LoginPage from './LoginPage'

describe(`LoginPage component`, () => {


  it('renders the LoginPage component by default', () => {
    const wrapper = shallow(<LoginPage />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })

})