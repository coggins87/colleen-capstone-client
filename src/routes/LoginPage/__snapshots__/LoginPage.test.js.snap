// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginPage component renders the LoginPage component by default 1`] = `
<section
  className="LoginPage"
>
  <h2
    className="page_header"
  >
    Log in
  </h2>
  <LoginForm
    onLoginRedirect={[Function]}
  />
</section>
`;
